pinot-plugins/pinot-stream-ingestion/pinot-pulsar/src/main/java/org/apache/pinot/plugin/stream/pulsar/PulsarPartitionLevelConnectionHandler.java

```diff
@@ -73,10 +73,10 @@ public PulsarPartitionLevelConnectionHandler(String clientId, StreamConfig strea
     }
   }
 
-  private Optional<Authentication> getAuthenticationFactory(PulsarConfig pulsarConfig) {
+  protected Optional<Authentication> getAuthenticationFactory(PulsarConfig pulsarConfig) {
```

Comment:
If this scope is changed just for testing, can we add @VisibleForTesting annotation

Reply:
Yes, I just changed it to test it. Will add the annotation.